packages/arb-token-bridge-ui/src/util/withdrawals/__tests__/fetchTokenWithdrawalsFromEventLogs.test.ts

```diff
@@ -7,17 +7,25 @@ import {
 
 describe('fetchTokenWithdrawalsFromEventLogs', () => {
   it('fetches no token withdrawals from event logs pre-nitro', async () => {
-    const result = await fetchTokenWithdrawalsFromEventLogs(
-      getQueryCoveringClassicOnlyWithoutResults()
-    )
+    // TODO: This is a temporary fix, when Event Logs are enabled for custom address
```

Comment:
Got it. Linking this with https://github.com/OffchainLabs/arbitrum-token-bridge/issues/1060